generate.py

```diff
@@ -264,6 +288,10 @@
             v['distance'] = next_distance
             v["departure"] = next_dep_time.isoformat()
 
+            desired_soc = soc_needed * (1 + vars(args).get("buffer", 0.1))
```

Comment:
Is this 0.1 (10%?) hardcoded or what does it represent? If so, it would be better to have this buffer in % in the config file